WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeLearnHowView.swift

```diff
@@ -0,0 +1,242 @@
+import SwiftUI
+
+/// View to display the learn how sheet of Blaze campaign creation
+///
+struct BlazeLearnHowView: View {
+    @Binding var isPresented: Bool
+
+    private let steps: [Step] = [.init(number: Localization.ChooseProduct.number,
+                                       title: Localization.ChooseProduct.title,
+                                       subtile: Localization.ChooseProduct.subtitle),
+                                 .init(number: Localization.CustomizeTargeting.number,
+                                       title: Localization.CustomizeTargeting.title,
+                                       subtile: Localization.CustomizeTargeting.subtitle),
+                                 .init(number: Localization.SetYourBudget.number,
+                                       title: Localization.SetYourBudget.title,
+                                       subtile: Localization.SetYourBudget.subtitle),
+                                 .init(number: Localization.QuickReview.number,
+                                       title: Localization.QuickReview.title,
+                                       subtile: Localization.QuickReview.subtitle),
+                                 .init(number: Localization.GoLive.number,
+                                       title: Localization.GoLive.title,
+                                       subtile: Localization.GoLive.subtitle)]
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            titleBlock
+
+            stepBlock
+        }
+    }
+}
+
+private extension BlazeLearnHowView {
+    var titleBlock: some View {
+        ZStack(alignment: .center) {
+            HStack {
+                Button(action: {
+                    isPresented = false
+                }, label: {
+                    Image(uiImage: .closeButton)
+                        .secondaryBodyStyle()
+                })
+
+                Spacer()
+            }
+            .padding(.horizontal, Layout.TitleBlock.closeButtonHorizontalPadding)
+
+            HStack {
+                Text(String(format: Localization.title, "Blaze"))
+                    .fontWeight(.semibold)
+                    .headlineStyle()
+            }
+        }
+        .padding(.top, Layout.TitleBlock.topPadding)
+    }
+
+    var stepBlock: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0) {
```

Comment:
This extra VStack is redundant, we should be able to apply the corner radius to the inner one.

Reply:
Good point. Removed in https://github.com/woocommerce/woocommerce-ios/pull/11583/commits/79e4dc8a28292333e53a9df5631de1adcaa9d4a5